### PR TITLE
Added two new feature files, updated model and step files

### PR DESCRIPTION
**Pull request by sujatagit**
- Added two new features
- Updated models - added shippingDetails function in the base
- Updated the step file
Screenshot 2024-11-29 at 9 16 02 PM
Screenshot 2024-11-29 at 9 17 39 PM

### CODE REVIEW

**Review by sujatagit — COMMENTED** [review]
Approved

File: cucumber.js [hunk 1]
```diff
@@ -5,7 +5,10 @@ console.log(`expected report file should include ${timestamp}`);
 
 let common = [
     // 'features/**/*.feature',                // Specify our feature files
```
Comment (anahita-m) [inline cucumber.js]: remove commented out code

File: models/Base.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const { Base } = require('./Base');
+// const { Base } = require('./Base');
```
Comment (anahita-m) [inline models/Base.js]: remove commented out code

File: cucumber.js (hunk 1 above)
Comment (sujatagit) [reply cucumber.js]: Resolved

**Review by anahita-m — APPROVED** [review]

**Pull request merged**